packages/builder/src/steps/invoke.test.ts

```diff
@@ -202,8 +202,11 @@ describe('steps/invoke.ts', () => {
         {
           target: ['Woot'],
           func: 'something',
-          args: ['foo', { bar: 'baz' }],
-          value: '1234',
+          args: [
```

Comment:
why were these args changed? having a nested argument/tuple arg is part of the test

Reply:
Changed them back, I did need to however add another argument as this was causing the test to fail saying it requires 3 arguments because of the argument encode check I added in this branch:

![image](https://github.com/usecannon/cannon/assets/30664234/3c7a186f-7dfd-4685-842a-af6bc2525a24)